bin/MadGraph5_aMCatNLO/cards/production/13TeV/WPrime_tb/WPrime_tb_proc_card.dat

```diff
@@ -0,0 +1,15 @@
+import model WPrimetb_UFO
+define bs = b b~
+define p = p bs
+define j = j bs
+define wp = wp+ wp-
+define ls = l+ l- vl vl~
```

Comment:
l+ doesn't include tau decays. you only want e+ and mu+ ?

Reply:
Right, we're trying to avoid tau decays

Reply:
How will you handle tau -> e or tau -> mu in the analysis ?

Reply:
Sorry for the late reply, I must have missed the notification for this comment. 

We didn't include taus because we don't want hadronic decays and don't know how to properly limit the tau decays to only leptonic decays. The plan now is to just make an estimate of the signal contamination produced by the taus, but if there is another card we can add, or some other method, to limit the taus to leptonic decays it would be preferable to add them.

Reply:
An alternative would be to limit the tau decay in P8 to e and mu. Or the filter after decay. Up to you. I am also happy to merge as it is. Just confrim.

Reply:
Let's go ahead and merge the cards as they are now and we'll just continue as originally planned.